crates/store/src/genesis/config/tests.rs

```diff
@@ -0,0 +1,16 @@
+use super::*;
+
+#[test]
+fn simple_01_works() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
nit: Would rename this test to express what it validates about the genesis file to some extent. It could become parameterized in the future if we had multiple files to test. Having the filename in the test name doesn't feel right / help the reader.